### ISSUE TITLE
Inference removal

### ISSUE THREAD

**Issue by barny116** [vdemichev/DiaNN#1066]
Hi, 
I noticed after looking closer at the report.log for my search that the following is disabled: "inference removal from fragment elution curves disabled." Is it typical for most studies for this feature to be disabled, in what instances is it necessary?
Thanks for your help!

**Comment by vdemichev** [comment]
Yes, that's perfectly fine and indicates that you are using the 'high-precision' quantification mode in DIA-NN 1.8.1.

Best,
Vadim
